unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PlayerPreview/PassportPlayerPreviewComponentView.cs

```diff
@@ -0,0 +1,5 @@
+using UnityEngine;
+
+public class PassportPlayerPreviewComponentView : MonoBehaviour, IPassportPlayerPreviewComponentView
```

Comment:
In order to follow our UI Components architecture, all our views should implement the `BaseComponentView` class.